test/test_logging_output_format.py

```diff
@@ -72,6 +72,25 @@ def generate_test_description():
     ))
     processes_to_test.append(name)
 
+    env_simple_escape_tokens = dict(os.environ)
+    # This custom output is to check that escape characters work correctly.
+    env_simple_escape_tokens['RCUTILS_CONSOLE_OUTPUT_FORMAT'] = '\t{name}\n\t{message}\n'
```

Comment:
I'm somewhat concerned that this test (and the one below) are going to do unexpected things and not work on Windows.  But I guess we'll find out when we run CI.

Reply:
Try running CI in https://github.com/ros2/rcutils/pull/426#issuecomment-1674122772